packages/web-components/src/slider/slider.options.ts

```diff
@@ -0,0 +1,16 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
```

Comment:
We should likely re-export the SliderOrientation from Foundation to ensure that everything can be included when bringing this in (no other exports needed).

Reply:
Would this look like:
```
import { SliderOrientation } from '@microsoft/fast-foundation';

/**
 * Applies orientation to the slider
 * @public
 */
export type SliderOrientation = ValuesOf<typeof SliderOrientation>;
```
Or should I define the options as I did in SliderSize, such as:
```
/**
 * SliderOrientation Constants
 * @public
 */
export const SliderOrientation = {
  horizontal: 'horizontal',
  vertical: 'vertical',
} as const;
```